protocol.md

```diff
@@ -317,8 +317,47 @@ The value of the  `Upload-Expires` header MUST be in
 
 ### Checksums
 
-This extension will define how to provide per file or per chunk checksums for
-uploaded files.
+Clients and servers MAY implement and use this extension to verify data
+integrity per chunk. In this case the server MUST add the `checksum` element to
+the `TUS-Extension` header.
+
+A client MAY include the `Content-MD5` header and its appropriate value in a
```

Comment:
What would be your (rough, just to get an idea what you would like to see changed/included) proposal?